jujugui/static/gui/src/app/init/test-endpoint-utils.js

```diff
@@ -558,12 +558,7 @@ describe('Application config handlers', () => {
         id: 'wordpress',
         pending: true,
         charm: charmUrl});
-      assert.equal(conn.messages.length, 1);
```

Comment:
I believe these counts must have been wrong before due to race conditions with other tests.